### PR TITLE
Feature/weos 1029

### PR DESCRIPTION
**Pull request by atoniaw**
WEOS-1029 and WEOS-1030

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   - Adds request entity to the customer
   - Adds request create handlers
   - Adds request update handlers

### CODE REVIEW

File: customer.go [hunk 1]
```diff
@@ -10,9 +11,8 @@ type Customer struct {
 	Name             string                    `json:"name"`
 	Notes            string                    `json:"notes"`
 	CSR              *weos.User                `json:"csr"`
-	Tickets          map[string]*Ticket        `json:"tickets"`
+	Requests         map[string]*Request       `json:"tickets"`
```
Comment (EmmanuelSammy) [inline customer.go:14]: should json be tickets?

File: value_objects.go [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
+	State     string  `json:"state"`
+	Zip       string  `json:"zip"`
+	CountryID *int    `json:"countryId"`
+	Country   string  `json:"country"`
+}
+
+type RequestTask struct {
+	Title       string    `json:"title"`
+	Description string    `json:"description"`
+	DueDate     time.Time `json:"dueDate"`
```
Comment (EmmanuelSammy) [inline value_objects.go:50]: should prob be an int

**Review by EmmanuelSammy — COMMENTED** [review]
Made some comments lmk if its okay or not

File: value_objects.go (hunk 2 above)
Comment (atoniaw) [reply value_objects.go:50]: No it's an actual date.  This differs from the other Task structure in that it has an actual due date.

**Review by akeemphilbert — APPROVED** [review]

**Pull request merged**